src/pages/Access/AccessPage.tsx

```diff
@@ -1,15 +1,9 @@
-// import { FunctionComponent } from "react";
```

Comment:
Now this page seems less useful? but maybe good to have it around in case we want this setup elsewhere.